redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/handler/CertRotateInRedbeamsHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.handler;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.eventbus.Event;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedbeamsRequest;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedbeamsSuccess;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerFailed;
+
+@Component
+public class CertRotateInRedbeamsHandler extends ExceptionCatcherEventHandler<CertRotateInRedbeamsRequest> {
```

Comment:
Please add UT for this handler

Reply:
what should I need to test?

Reply:
I'd add a test for checking if the doAccept function returns the correct event. Even if it's not a very useful test currently it can happen that actual logic is added later to the function that changes the returned event and its harder to miss creating proper UTs if a test already exists for the function, that's why i think it's a always a good idea to test "skeleton" code as well. 

Also I usually add a test for handler's selector function but that's not the general case for every handler so I'd leave that up to you.

Reply:
done

Reply:
thanks!